fetch.bs

```diff
@@ -301,6 +301,9 @@ following <a for=struct>items</a>: [[RESOURCE-TIMING]] [[NAVIGATION-TIMING]]
  <dt><dfn export for="response body info" id="fetch-timing-info-decoded-body-size">decoded
  size</dfn> (default 0)
  <dd>A number.
+ <dt><dfn export for="response body info" id="fetch-timing-info-response-status">response
+ status</dfn> (default 0)
```

Comment:
One minor thing here that I notice is incorrect for some other fields as well is that `fetch.bs` shouldn't wrap inside phrasing elements such as `<dfn>` so you can more easily search for the terms inside of them.